### ISSUE TITLE
With Yeti's rule active, Ice Age does nothing against Robot

### ISSUE THREAD

**Issue by JuniorUdale** [TerryCavanagh/diceydungeons.com#2884]
![image](https://user-images.githubusercontent.com/32357485/174817314-8a05e9fc-4922-411f-a4d7-c0ec659a23bc.png)

With 2 freeze, and "freeze all" from Ice Age active, a 6 turned into a 4. I'm not sure what the intended behaviour here is in the first place - "freeze all" is weird in the context of PU.

**Comment by ncrecc** [comment]
i'm pretty sure it nudges all your dice every time you calculate

**Comment by TerryCavanagh** [comment]
Yep, that's right! The intended behaviour is to nudge your dice with each calculate. It looks like from the screenshot that that didn't happen?

**Comment by JuniorUdale** [comment]
The 2 stacks of Freeze? worked (a 6 became a 4), but the Freeze All? had no effect.

**Comment by TerryCavanagh** [comment]
Aha, thank you - I get it now.

**Comment by ncrecc** [comment]
i did some testing and it seems like ``Rules.setalternate(ICE)`` correctly alts the "frozen" status but *not* the "frozen all" status. the same issue may apply with Wicker Man rule - with ``Rules.setalternate(FIRE)`` alting "burn" but not "burn all", though i haven't tested that yet
